Sources/tinys3/model/Multipart/MultipartUploadOperation.swift

```diff
@@ -102,6 +133,15 @@ class MultipartUploadOperation: NSObject, RequestPerformer {
             data: try await file[range]
         )
 
+        if let existingPart, existingPart.eTag == "\"\(md5Hash(data: part.data))\"" {
```

Comment:
For whatever reason, the `ETag` field in S3 responses contains literal quotes around its value. So I had to add them to the locally-computed `md5Hash` before doing the `==`.